src/api/auth/Account.service.js

```diff
@@ -44,7 +57,7 @@ export function fetchUserContext() {
 }
 
 export function isLoggedIn() {
-  if (localStorage.getItem(AUTH_TOKEN_KEY)) {
+  if (localStorage.getItem(AUTH_TOKEN_KEY) && isAccessTokenValid()) {
```

Comment:
It seems `isAccessTokenValid()` also checks for the existence of the token. Can the condition be reduced to just `if isAccessTokenValid()`?